blog/content/post/2023-10-22-liquid-cooling.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Upgrading your gears with liquid cooling
+abstract: 'Abstract first sentence.
+          Abstract second sentence.
+          Abstract third sentence.'
+cover: /covers/image-file.png
+author: michal.zygowski
+layout: post
+published: true
+date: 2023-10-22
+archives: "2023"
+
+tags:
+  - tag 1
+  - tag 2
+categories:
+  - Firmware
+  - IoT
+  - Miscellaneous
+  - OS Dev
+  - App Dev
+  - Security
+  - Manufacturing
+
+---
+
+## Introduction
+
+A time comes when you want to upgrade your gears in your desktop, say a CPU.
+Either your use case changes or you have different needs, which force you to
+look for a different, more beefy CPU. In general CPU exchange is limited to
+swapping the CPU in the socket. But is that all?
+
+## Upgrading your cooling solution
+
+Depending on your setup, your old CPU may have behaved well with the current
+cooling you have. It was selected to satisfy the needs of your old CPU by
+providing enough heat dissipation. But a CPU upgrade  may cause your cooling
+solution to be insufficient, especially if the performance gap between the old
+and new CPU is significant. In this blog post I will show you a short story of
+upgrading from Intel Core i5-12600K to Intel Core i9-14900K and a standard
+fan+radiator solution to liquid cooling solution and what difficulties you may
+encounter during such journey.
+
+### Choosing an appropriate model
+
+Choosing a liquid cooling model is not that simple task. First of all you have
+to take the chassis dimensions into consideration. So it is necessary to take
+a tape measure and note the dimensions of the chassis sides with mounting
+holes. Typically this is a top or front side (at least in my case). These
+should be prepared to fit 120 mm wide fans or radiators by standard. However,
+depending on the height on the chassis, the number of fans you will be able to
+fit will be different. In my case the choice was as follows:
+
+- take liquid cooling with 2x120mm fans (either on top or front side)
+- take liquid cooling with 3x120mm fans (only front side)
+
+Thankfully my front side can fit 3x120mm and considering the performance gap
+in my CPUs, I decided to go with 3x120mm solution MSI MAG Coreliquid 360R v2.
+But little did I know back then that mounting it will not be an easy task.
+
+### Watching liquid cooling mounting tutorials
+
+I may be a BIOS/firmware developer, but little do I know about advanced PC
+assembling like liquid cooling assembly. Before I could start playing with it,
+I had to watch a few tutorials how to NOT mount the liquid cooling to make it
+perform at its best. Personally I recommend the following in general:
+
+- [The Gamer's Nexus' guide](https://www.youtube.com/watch?v=BbGomv195sk)
+- [The Provoked Prawn's guide](https://www.youtube.com/watch?v=qQcyYHGtArs)
```

Comment:
I would say the embed version would be better.

Reply:
https://github.com/3mdeb/news-and-ideas/pull/545/commits/2b5b9cc842c5f672f7fc60bacf4139b5fa6b19d6#diff-bd4456b5082aa26d9ea51ebc4ce2ed44da6fd4f97d01097d5221951114a52590L69-R79